fastlane/screenshots/README.md

```diff
@@ -0,0 +1,129 @@
+# How to generate & update Play Store Screenshots
+
+The screenshots used for the Play Store come in multiple device sizes (phone vs tablet) and many locales (currently 16 different languages), so it would be unreasonable to expect designers to generate all the variants by hand every time we want to update those screenshots. For this reason, the generation of those Play Store screenshots are automated with fastlane.
```

Comment:
```suggestion
The screenshots used for the Play Store come in multiple device sizes (phone vs tablet) and many locales (currently 16 different languages), so it would be unreasonable to expect designers to generate all the variants by hand every time we want to update those screenshots. For this reason, the generation of those Play Store screenshots is automated with Fastlane.
```